website/src/components/TreeView/DictionaryTree.js

```diff
@@ -209,7 +209,7 @@ const FileNode = ({ fileDef, onExpandStateChange }) => {
         className={css`
           &::before,
           &::after {
-            left: -4px !important;
+            _left: -4px !important;
```

Comment:
what's `_left`?

Reply:
meant to remove this, thanks @hlminh2000

Reply:
sometimes use the `_` to make it invalid instead of deleting/comment :)

Reply:
lol I was almost excited to learn some new css :P

Reply:
fixed @hlminh2000